lib/Perl/Critic/Policy/TooMuchCode/ProhibitDuplicateLiteral.pm

```diff
@@ -12,8 +14,21 @@ sub violates {
     my %firstSeen;
     my @violations;
 
+    my @allowed_strings;
+
+    my $allowed_strings_config = $self->__get_config->get('allowed_strings');
+
+    if (defined $allowed_strings_config) {
+        my $allowed_strings_parser = PPI::Document->new(\$allowed_strings_config);
+
+        for my $quoted_token (@{$allowed_strings_parser->find('PPI::Token::Quote') ||[]}) {
+            push @allowed_strings, $quoted_token->string;
+        }
+    }
```

Comment:
parsing "a list of strings" like this seems a bit unconventioal to me, and that would means we need to write some documentation just to describe the basic syntax of this list. I wonder if there is anything to steal from the `Perl::Critic` tframework...

I guess there is probably some way to declare that we want "a list of string literals that could contain whitespaces"  inside `supported_parameters`. Similar to what's been done here:

https://github.com/gugod/Perl-Critic-TooMuchCode/blob/master/lib/Perl/Critic/Policy/TooMuchCode/ProhibitExtraStricture.pm#L12

Reply:
> parsing "a list of strings" like this seems a bit unconventioal to me

I agree, though I did not find any other good or pre-existing way to handle a list of "strings-with-spaces" :disappointed: 

Therefore I ended up with the current quoted strings approach as a first approximation. I find quoted strings to be the closest representation of what I expect to end up as list items, and I hope this would lead to the least amount of surprises.